ckanext/switzerland/helpers/mapping.yaml

```diff
@@ -53,7 +58,16 @@ PDF:
 PNG:
   - 'png'
 RDF:
-  - 'sparql-...'
+  - 'rdf'
+  - 'rdf+xml'
```

Comment:
I would show separately RDF/XML, as you did for N-Triples and turtle, since this is also one of the RDF serialization format.

Reply:
I thought about it, but I didn't want to make a change that would affect existing data without clearing it with the BFS first. `RDF` by itself is also a valid file type in the EU vocabulary: https://op.europa.eu/en/web/eu-vocabularies/concept/-/resource?uri=http://publications.europa.eu/resource/authority/file-type/RDF
``` File ext:
Identifiers
File ext: .rdf
IANA MIME: application/rdf+xml
```
So I think it should be OK to leave it like this. Or do you think I should ask Michelle and Pascal about changing RDF->RDF XML?

Reply:
I would not mix them, since it is not the same. Look at the EU portal, they also give RDF_XML https://op.europa.eu/en/web/eu-vocabularies/concept/-/resource?uri=http://publications.europa.eu/resource/authority/file-type/RDF_XML

Reply:
In the "Format" facet one can find "RDF XML"

Reply:
> I would not mix them, since it is not the same. At the EU portal, they also give RDF_XML https://op.europa.eu/en/web/eu-vocabularies/concept/-/resource?uri=http://publications.europa.eu/resource/authority/file-type/RDF_XML

Yes, I have seen this.

> In the "Format" facet one can find "RDF XML"

Which Format facet do you mean?

Reply:
I mean in the open data facet https://data.europa.eu/data/datasets?locale=en&format=RDF_XML&page=1

Reply:
There are also 2.5x as many datasets there for "RDF". ;) https://data.europa.eu/data/datasets?locale=en&format=RDF&page=1

Reply:
not sure if I am understanding this correctly, pretty sure I don't :D 
But when we set the format to `rdf+xml` it will be displayed with an XML-icon and I am not sure if that is what was expected of adding this format? Happy to discuss this as well :)